### ISSUE TITLE
Erasing an object does not delete associated fields on relations (Activity Targets specifically)

### ISSUE THREAD

**Issue by GP-S** [twentyhq/twenty#2787]
## Bug Description

On 0.1.5 after delzeting a new data model, twenty seems to have broke : I can't display tables anymore

![image](https://github.com/twentyhq/twenty/assets/9041351/52b38db9-2b0c-4377-86e2-2e1799f07505)

## Step to reproduce:

- Create new simple data
- Diable old field except name and create data
- Going back to home page
- Then delete data model (I wanted to change the neme but it seems you can't)
- Got broken interface

## Technical inputs

- Browser : firefox 120
- OS : W11

```
useFindManyObjectRecords for "views" error : ApolloError: Response not successful: Received status code 500 main.0a894f36.js:1:1123740
    fd https://app.twenty.com/static/js/main.0a894f36.js:1
    onError https://app.twenty.com/static/js/main.0a894f36.js:1
    handleErrorOrCompleted https://app.twenty.com/static/js/main.0a894f36.js:1
```
```
Uncaught (in promise) ApolloError: Response not successful: Received status code 500
    t https://app.twenty.com/static/js/main.0a894f36.js:1
    error https://app.twenty.com/static/js/main.0a894f36.js:1
    w https://app.twenty.com/static/js/main.0a894f36.js:1
    k https://app.twenty.com/static/js/main.0a894f36.js:1
    error https://app.twenty.com/static/js/main.0a894f36.js:1
    c https://app.twenty.com/static/js/main.0a894f36.js:1
    w https://app.twenty.com/static/js/main.0a894f36.js:1
    k https://app.twenty.com/static/js/main.0a894f36.js:1
    error https://app.twenty.com/static/js/main.0a894f36.js:1
    r https://app.twenty.com/static/js/main.0a894f36.js:1
    r https://app.twenty.com/static/js/main.0a894f36.js:1
    error https://app.twenty.com/static/js/main.0a894f36.js:1
    w https://app.twenty.com/static/js/main.0a894f36.js:1
    k https://app.twenty.com/static/js/main.0a894f36.js:1
    error https://app.twenty.com/static/js/main.0a894f36.js:1
    error https://app.twenty.com/static/js/main.0a894f36.js:1
    w https://app.twenty.com/static/js/main.0a894f36.js:1
    k https://app.twenty.com/static/js/main.0a894f36.js:1
    error https://app.twenty.com/static/js/main.0a894f36.js:1
    w https://app.twenty.com/static/js/main.0a894f36.js:1
    k https://app.twenty.com/static/js/main.0a894f36.js:1
    error https://app.twenty.com/static/js/main.0a894f36.js:1
    nd https://app.twenty.com/static/js/main.0a894f36.js:1
    u https://app.twenty.com/static/js/main.0a894f36.js:1
    u https://app.twenty.com/static/js/main.0a894f36.js:1
    a https://app.twenty.com/static/js/main.0a894f36.js:1
main.0a894f36.js:1:912603

```

**Comment by GP-S** [comment]
@FelixMalfait I'm currently on the Saas version, Ican give you access if needed.

**Comment by GP-S** [comment]
> Hi @GP-S,
> 
> Could you use the support button on the bottom left of the page to contact us. We can help you from there (without disclosing your user ids here)

I pinged the suppot chat with the issue link :)

**Comment by charlesBochet** [comment]
I have fixed your workspace! Thanks for raising the issue, the erase feature seems to be buggy indeed, let's keep this ticket open to track bug fix

@Weiko FYI

**Comment by charlesBochet** [comment]
Resolved since v0.3.2
